src/internal/EnhancedButton.js

```diff
@@ -289,13 +289,6 @@ class EnhancedButton extends Component {
       outline: 'none',
       fontSize: 'inherit',
       fontWeight: 'inherit',
-      /**
-       * This is needed so that ripples do not bleed
-       * past border radius.
-       * See: http://stackoverflow.com/questions/17298739/
-       * css-overflow-hidden-not-working-in-chrome-when-parent-has-border-radius-and-chil
-       */
-      transform: disableTouchRipple && disableFocusRipple ? null : 'translate(0, 0)',
```

Comment:
@oliviertassinari we use FloatingActionButton with FontIcon that have rotate on hover, (in this case border radius not working)